spec/lib/embed/media_tag_spec.rb

```diff
@@ -30,11 +30,11 @@
     end
 
     it 'includes a data-src attribute for the dash player' do
-      expect(subject).to have_css('[data-src]', count: 2, visible: false)
+      expect(subject).to have_css('[data-src]', count: 3, visible: false)
     end
 
     it 'includes a data attribute that includes the url to check the users auth status' do
-      expect(subject).to have_css('video[data-auth-url]', count: 2, visible: false)
+      expect(subject).to have_css('video[data-auth-url]', count: 3, visible: false)
```

Comment:
One of the video objects in the original fixture was location restricted, so effectively no-download.  That resulted in some other tests breaking.  I felt the easiest way to make the fixture work for the various tests was to add another file to the fixture that was NOT download restricted.  Hence this change.